kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -43,9 +43,18 @@
         <div class="text-small flex-container flex-center"
              ng:if="ctrl.saving || dropIntoCollectionSaving">Adding…</div>
 
+        <div class="text-small flex-container flex-center"
+             ng:if="ctrl.removing">Removing…</div>
+
+        <button class="node__action clickable hover-child"
+                title="Remove selected images from this collection"
+                ng:if="!ctrl.editing && ctrl.hasImagesSelected && !ctrl.saving && !ctrl.removing"
```

Comment:
Could it? Surely it needs to be updated if any of `hasImagesSelected`, `editing` etc gets updated?